### PR TITLE
NetBox 2.9 release

### LINKED ISSUE #112: Support for NetBox 2.9.X
Opening this issue so there's something to track.

From the role's perspective, I don't know that there's much that requires updating.

https://github.com/netbox-community/netbox/blob/develop/docs/release-notes/version-2.9.md#configuration-changes

Those can be handled via `netbox_config`.

### PR DESCRIPTION
**Pull request by lae**
This introduces feature support for NetBox 2.9. If you can test this branch out by upgrading an existing installation (or copy of one), please do so and report back. Confirmations of successful upgrades will result in a quicker merge (as I'm having some test suite issues I need to resolve currently).

To deploy NetBox 2.9, use `ansible-galaxy` to install this branch:

```
ansible-galaxy install https://github.com/lae/ansible-role-netbox/archive/feature/netbox-2.9.tar.gz,v1.1.0-pre,lae.netbox --force
```

## Changes

Not very many significant ones as far as I can tell. If I'm missing anything, please leave a comment.

* `RQ_DEFAULT_TIMEOUT` under redis config ([2.9.4 changelog](https://github.com/netbox-community/netbox/blob/develop/docs/release-notes/version-2.9.md#v294-2020-09-23))

Closes #112

### CODE REVIEW

**Comment by ironiq** [comment]
Just one note: Netbox 2.9+ requires Redis v4.x or later. The default settings in the role davidwittman.redis wants to install 2.8, this is not overridden here. Redhat 7 contains 3.2, Ubuntu 16.04 contains 3.0, so i think that should be mentioned in the README.

**Comment by tyler-8** [comment]
Just used this to install in a test instance, worked great!

**Comment by lae** [comment]
@tyler-8 Can you do me a favour and review the PR changes and approve it?

File: templates/configuration.py.j2 [hunk 1]
```diff
@@ -22,18 +22,25 @@ DATABASE = {
     'HOST':  '{{ netbox_database_host }}',
     'PORT': '{{ netbox_database_port }}',
 {% else %}
-    {% if netbox_database_password is defined %}
+{%      if netbox_database_password is defined %}
```
Comment (tyler-8) [inline templates/configuration.py.j2:25]: This indentation style for Jinja is a little weird to me, just never seen it before - but functionally it's fine.

Comment (lae) [reply templates/configuration.py.j2:25]: I'm using it here to get rid of the excessive whitespace in the resulting file whilst retaining readability of nested statements. But yeah, it's also kinda weird to me.

**Comment by lae** [comment]
https://travis-ci.org/github/lae/ansible-role-netbox/builds/741530433

(still trying to figure out why travis integration is broken)

**Review by tyler-8 — APPROVED** [review]
I like the handling of the renamed configuration param. Looks good!

File: templates/configuration.py.j2 (hunk 1 above)
Comment (tyler-8) [reply templates/configuration.py.j2:25]: Looks like maybe you can add `#jinja2: lstrip_blocks: True` to the top of templates to maybe help reduce the whitespace, food for thought for the future

**Pull request merged**